qiskit/circuit/library/n_local/n_local.py

```diff
@@ -306,6 +306,86 @@ class NLocal(BlueprintCircuit):
 
         The :func:`.n_local` function constructs a functionally equivalent circuit, but faster.
 
+    Examples:
```

Comment:
Some examples are a good idea 🙂 The `n_local` function will take over as main object and `NLocal` is going to be deprecated in v2.1. Could you move these example to the `n_local` function instead? They might need some minor adjustments.